traitsui/editor.py

```diff
@@ -129,6 +135,15 @@ class Editor(HasPrivateTraits):
     #: The current editor invalid state status:
     invalid = Bool(False)
 
+    #: A set to track values being updated to prevent infinite recursion.
+    _no_trait_update = Set()
+
+    #: A list of all values synchronized to.
```

Comment:
Please could you indicate either in the comment or in the trait type definition what the contents of the tuples are? (What their expected length is, and what the contents of the fields are.)